pandas/tests/io/parser/test_parsers.py

```diff
@@ -100,3 +102,51 @@ def read_table(self, *args, **kwds):
         kwds = kwds.copy()
         kwds['engine'] = self.engine
         return read_table(*args, **kwds)
+
+
+class TestUnsortedUsecols(object):
+    def test_override__set_noconvert_columns(self):
```

Comment:
hah you are basically mocking the parser here....ok. I didn't mean go that far, but this is ok.